mavionics/src/modules/RtcModule.js

```diff
@@ -54,42 +63,49 @@ class RtcModule {
       stream: this.stream
     });
 
+    this.peer._debug = console.log
+
+
+
     this.inMessages.onSnapshot(querySnapshot => {
       querySnapshot.forEach(doc => {
-        this.p.signal(doc.data());
+        this.peer.signal(doc.data());
       });
     });
 
-    this.p.on("signal", data => this.outMessages.add(data));
-    this.p.on("stream", this.onStream);
-    this.p.on("data", data => {
+    this.peer.on("signal", data => this.outMessages.add(data));
+    this.peer.on("stream", this.onStream);
+    this.peer.on("data", data => {
       console.warn(data);
       this.onMessage(JSON.parse(data));
     });
 
     return new Promise((resolve, reject) => {
-      this.p.on("error", function(err) {
-        console.log("error", err);
+      this.peer.on("error", function(err) {
+        console.log("RtcModule.js peer error: ", err);
         reject(err);
       });
 
-      this.p.on("connect", () => {
+      this.peer.on("connect", () => {
         this.dbRef.update({ status: "connected" });
         resolve();
       });
     });
   }
 
   sendMessage(data) {
-    if (this.p !== null) this.p.send(JSON.stringify(data));
+    console.log("RtcModule.js sendMessage ", JSON.stringify(data));
+    if (this.p !== null) this.peer.send(JSON.stringify(data));
   }
 
   disconnect() {
-    this.p.destroy();
+    console.log("RtcModule.js sendMessdisconnectge ");
+    this.peer.destroy();
   }
 
   isConnected() {
-    return this.p && this.p.connected;
+    console.log("RtcModule.js isConnected");
+    return this.p && this.peer.connected;
```

Comment:
this.p should be this.peer I assume?

Reply:
Could be why this has been chrashing for me ;)

Reply:
Could be why this has been chrashing for me ;)